### DISCUSSION TITLE
Add text to "See inside" button in order to give info about downloading and editing projects

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#5726] [Ideas]
New Scratchers can't know that they can actually download projects or how to download them as there are no info on the project pages. 

My suggestion is adding a (non bold, small font) text "to edit or download" under the "See inside" text in the blue button.

**Comment by CST1229** [comment]
Alternatively, it could add an additional "Download" button, which would also be useful for experienced users (to download a project quickly).

**Comment by DNin01** [comment]
This seems like something Scratch should help tutorialize - most new Scratchers won't have Scratch Addons and optional addons aren't the best fit for tutorials/guides. Feel free to make a suggestion to Scratch.

We could, however, think about making a download button addon.
